### DISCUSSION TITLE
Is there an ETA for version 2

### DISCUSSION THREAD

**Discussion by Akhil-CM** [mattmc3/antidote#198] [General]
Some time ago, I stumbled upon this repo and saw the note on the upcoming version 2/2.0. I am waiting to try it after 2.0 is released.

Btw I hope my question doesn't lead to any developer burnout. No ETA is completely fine.

**Comment by mattmc3** [comment]
I welcome the question - glad you asked. I don't have a specific ETA, but these past few months have been too busy with other things and I want to make sure I have some time after a bigger release to address any bug reports. I'm tentatively planning to wrap this up by the end of this year. Are there any features you can't live without that are causing you to wait? The 2.0 release shouldn't break anything in 1.9 - just changing some defaults and adding features.

**Comment by Akhil-CM** [comment]
Thanks for the answer.

I installed and set it up already.
